resources/definitions/creality_ender3v3ke.def.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": 2,
+    "name": "Creality Ender-3 V3 KE",
+    "inherits": "creality_base",
+    "metadata":
```

Comment:
sorry for the question, newbie here. but this definition gets the default for `machine_gcode_flavor` which is Marlin. Considering this printer ships with Klipper, does it need to be changed?